src/app/settings/premium.component.html

```diff
@@ -16,6 +16,9 @@ <h1>{{'goPremium' | i18n}}</h1>
             {{'premiumSignUpTwoStep' | i18n}}
         </li>
         <li>
+            <i class="fa fa-check text-success fa-il" aria-hidden="true"></i>
+            {{'premiumSignUpEmergency' |i18n}}
+        </li>
```

Comment:
We are missing an opening tag here.

Reply:
Sorry, but I did not understand what you meant.
I have opened the li tags and closed it as well. Which other tag do I have to include?
Thanks

Reply:
@vachan-maker , you're missing the opening `<li>` tag for the next list item below your change. You have a closing `</li>` tag noted where Oscar added this note, but then if you look below, there's no opening `<li>` tag for the next closing `</li>` tag in the DOM for `{{'premiumSignUpReports' | i18n}}`